### PR TITLE
PSR-219 Added Upload too many rows page

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

File: app/services/MemberDetailsUploadValidator.scala [hunk 1]
```diff
@@ -164,9 +167,9 @@ class MemberDetailsUploadValidator @Inject()(
           case nameDOBFormProvider.firstName => ValidationErrorType.FirstName
           case nameDOBFormProvider.lastName => ValidationErrorType.LastName
           case nameDOBFormProvider.dateOfBirth => ValidationErrorType.DateOfBirth
-          case dobDayKey => ValidationErrorType.DateOfBirth
-          case dobMonthKey => ValidationErrorType.DateOfBirth
-          case dobYearKey => ValidationErrorType.DateOfBirth
+          case `dobDayKey` => ValidationErrorType.DateOfBirth
```
Comment (kberec) [inline app/services/MemberDetailsUploadValidator.scala:170]: As a matter of interest, what is the reason for this change?

**Review by kberec — APPROVED** [review]

File: app/services/MemberDetailsUploadValidator.scala (hunk 1 above)
Comment (westwater) [reply app/services/MemberDetailsUploadValidator.scala:170]: it makes it pattern match against the value of `dobDayKey` defined above instead of assigning the value to `dobDayKey` to be used in the pattern matching function

**Review by kberec — APPROVED** [review]
